src/main/java/com/kodilla/ecommercee/domain/dto/OrderDto.java

```diff
@@ -0,0 +1,15 @@
+package com.kodilla.ecommercee.domain.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDate;
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
+public class OrderDto {
+    private Long id;
+    private Long userId;
+    private LocalDate orderDate;
+}
```

Comment:
I've been thinking about this for a long time and I don't really know what should be here. Maybe we'll add here Long cartId (or CartDto) as well, so as not to make two Dtos . I think, each order has a cart number.  

We must talk about some basic things. Maybe let everyone think carefully about how they see this whole project and we'll agree on the details in a chat or on a video

Reply:
i'd keep it separate, it makes the code easier to read & know what is what

Reply:
I'd also keep it seperate as our mentor said because these are two different Dtos. But I don't know if there should be more fields in OrderDto.

Reply:
I think it's a minor thing, we have to move forward with the project. You can merge it